lib/evm_database_ops.rb

```diff
@@ -89,7 +89,8 @@ def self.restore(db_opts, connect_opts = {})
     #   :username => 'samba_one',
     #   :password => 'Zug-drep5s',
 
-    uri = with_mount_session(:restore, db_opts, connect_opts) do |database_opts, _session, _remote_file_uri|
+    uri = with_mount_session(:restore, db_opts, connect_opts) do |database_opts, session, _remote_file_uri|
+      database_opts[:local_file] = session.download(database_opts[:local_file], connect_opts[:remote_file_name]) if session
```

Comment:
I think `connect_opts[:remote_file_name]` here can just be the yielded `remote_file_uri`